### ISSUE TITLE
[Frontend] Resolve accessibility violation in KYC Submission Form

### ISSUE THREAD

**Issue by emdevelopa** [emdevelopa/Stellar_Payment_API#1373]
Description
This task involves UX enhancement for the KYC Submission Form module.
The goal is to resolve accessibility violation in KYC Submission Form to improve the platform's styling, user interactions, and overall accessibility.

Requirements and context

- Must be secure, tested, and documented
- Adhere to the Drips Wave design and UI/UX standards
- Ensure compatibility across desktop and mobile browsers
- Specifically focused on frontend UX enhancement and responsiveness

Suggested execution

Fork the repo and create a branch
git checkout -b feature/fe-resolve-accessibility-violation-in-kyc-submission-form
Implement changes

Review existing component in KYC Submission Form
Apply changes: Resolve accessibility violation in KYC Submission Form
Use clean CSS or tailwind variables for styling
Maintain state transitions and visual feedback
Test and commit

Test mobile responsiveness and interactive states
Check accessibility (a11y) using standard audits
Include screenshots or gifs in the PR
Example commit message
feat(frontend): resolve accessibility violation in KYC Submission Form

Guidelines

Complexity: Medium (150 points)
Assignment required before starting
PR description must include: Closes #[issue_id]

**Comment by Demilade10** [comment]
@Demilade10 has applied to work on this issue as part of the **Stellar Wave Program**'s 8th wave.



> Hi I can fix this


ℹ️ **Repo Maintainers**: To accept this application, [review their application](https://www.drips.network/wave/maintainers/issues/aade26b9-2ce9-480d-9218-bd9bab555e51) or assign @Demilade10 to this issue.

**Comment by Emoji-dot** [comment]
@Emoji-dot has applied to work on this issue as part of the **Stellar Wave Program**'s 8th wave.



> I would like to work on this issue 


ℹ️ **Repo Maintainers**: To accept this application, [review their application](https://www.drips.network/wave/maintainers/issues/aade26b9-2ce9-480d-9218-bd9bab555e51) or assign @Emoji-dot to this issue.

**Comment by OlaBakare** [comment]
@OlaBakare has applied to work on this issue as part of the **Stellar Wave Program**'s 8th wave.



> Please, assign me this issue.
> I can handle it.
> Thank you.


ℹ️ **Repo Maintainers**: To accept this application, [review their application](https://www.drips.network/wave/maintainers/issues/aade26b9-2ce9-480d-9218-bd9bab555e51) or assign @OlaBakare to this issue.

**Comment by Sajjadd-dev** [comment]
@Sajjadd-dev has applied to work on this issue as part of the **Stellar Wave Program**'s 8th wave.



> Hello, I’d like to contribute to this issue and would appreciate the opportunity to work on it. Thank you.


ℹ️ **Repo Maintainers**: To accept this application, [review their application](https://www.drips.network/wave/maintainers/issues/aade26b9-2ce9-480d-9218-bd9bab555e51) or assign @Sajjadd-dev to this issue.

**Comment by rahmanabimbs** [comment]
@rahmanabimbs has applied to work on this issue as part of the **Stellar Wave Program**'s 8th wave.



> Hello! I’m eager to contribute to this project. I have hands-on experience developing and maintaining production-level applications and enjoy tackling well-defined tasks. My focus is on writing clean, maintainable code, communicating effectively, and consistently delivering high-quality work on schedule. I’m enthusiastic about the chance to collaborate and add value to the team.


ℹ️ **Repo Maintainers**: To accept this application, [review their application](https://www.drips.network/wave/maintainers/issues/aade26b9-2ce9-480d-9218-bd9bab555e51) or assign @rahmanabimbs to this issue.
